src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -87,7 +87,20 @@ public InternalProjectConfiguration(ProjectConfiguration config) {
             getReleaseConfiguration().setSkipSigning(true);
         }
         setJavaStaticLibs(System.getProperty("javalibspath")); // this can be safely set even if null. Default will be used in that case
-        setJavaFXStaticSDK(System.getProperty("javafxsdk"));  // this can be safely set even if null. Default will be used in that case
+        String javafxStaticSdkPath = System.getenv("JAVAFX_STATIC_SDK_PATH");
+        if (javafxStaticSdkPath != null) {
+            Path lib = Path.of(javafxStaticSdkPath, "lib");
+            if (!Files.exists(lib) || !Files.isDirectory(lib)) {
+                throw new IllegalArgumentException("Error: " + lib + " is not a valid SDK path.\n" +
+                        "Make sure you provide a valid JAVAFX_STATIC_SDK_PATH or unset it");
+            }
+            if (!Files.exists(lib.resolve("javafx.base.jar")) ||
+                    !Files.exists(lib.resolve("javafx.graphics.jar"))) {
+                throw new IllegalArgumentException("Error: " + lib + " doesn't contain required JavaFX jars.\n" +
+                        "Make sure you provide a valid JAVAFX_STATIC_SDK_PATH or unset it");
+            }
+        }
+        setJavaFXStaticSDK(javafxStaticSdkPath);  // this can be safely set even if null. Default will be used in that case
```

Comment:
Does this avoid a checksum check?

Reply:
The checksum check is only done when downloading from `FileDeps::resolvePath`:

```
        if (Files.exists(Objects.requireNonNull(path))) {
              return path;                                                // -> existing path
        }
        if (!setupDependencies()) {                           // -> download and check
        ...
        }
        
```